rust/integration-tests/src/models/queryable_models.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(clippy::extra_unused_lifetimes)]
+
+use bigdecimal::BigDecimal;
+use diesel::{Identifiable, Insertable, Queryable};
+use field_count::FieldCount;
+use processor::schema::{events, fungible_asset_activities, token_activities_v2};
+use serde::{Deserialize, Serialize};
+/**
+* Event model
+* this is created // b/c there is inserated_at field which isn't defined in the Event struct, we can't just load the events directly without specifying the fields.
+        // TODO: make this more generic to load all fields, then we should be able to run tests for all processor in one test case.
+
+*/
+#[derive(Clone, Debug, Deserialize, FieldCount, Identifiable, Insertable, Serialize, Queryable)]
+#[diesel(primary_key(transaction_version, event_index))]
+#[diesel(table_name = events)]
+pub struct Event {
```

Comment:
this looks good! Also this can benefit write query building if we need custom reading endpoint.  

another way i was thinking about is raw query like `select row_to_json(table_name) from table_name` and define `JsonResult`. And serialize it into expected db file.

Reply:
this can lower the complexity of the work here.

Reply:
oh yeah, we can improve this as we add more processors and tables!